packages/contracts-bedrock/test/L1/L1CrossDomainMessenger.t.sol

```diff
@@ -228,6 +278,20 @@ contract L1CrossDomainMessenger_Paused_Test is L1CrossDomainMessenger_TestInit {
         assertTrue(l1CrossDomainMessenger.paused());
         assertEq(l1CrossDomainMessenger.paused(), superchainConfig.paused(address(0)));
     }
+
+    /// @notice Tests paused state when systemConfig is zero address.
+    function test_pause_zeroSystemConfig_reverts() external {
```

Comment:
Another weird test, this isn't a behavior we actually want, it just shows that if you provide a system config that isn't actually a system config, then any functions that we depend on will fail. If we accept this test then there are a million other tests we should be writing. We probably need to tweak the prompt a bit so it doesn't go down this route.